### DISCUSSION TITLE
Pupil tracking on SLEAP

### DISCUSSION THREAD

**Discussion by LordKelv** [talmolab/sleap#1706] [Help!]
Hello SLEAP community!

I'm looking to track eye movements and pupil dilation on a head fixed mouse.  Would this be feasible in SLEAP?

Many thanks,
Kelvin

**Comment by Marielamend** [comment]
Hello @LordKelv!,

You can definitely use SLEAP to track this, here is a paper that features [pupil tracking using pose estimation](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10168383/ ). If you have any other specific questions about your workflow just let us know!

Best,
Mariela
